pkg/Microsoft.NETFramework.Compatibility/Microsoft.NETFramework.Compatibility.pkgproj

```diff
@@ -44,16 +41,20 @@
     <LibraryPackage Include="System.ServiceModel.Http" />
```

Comment:
Unfortunately the ServiceModel packages need to remain stable versions as they build outside of corefx. We cannot depend on prerelease versions of those